docs/api-reference/attribute-manager.md

```diff
@@ -13,29 +13,12 @@ Limitations:
   There are currently no provisions for only invalidating a range of
   indices in an attribute.
 
-## Static Methods
```

Comment:
`setDefaultLogFunctions ` is actually public - Just added it a couple of days ago. My mistake...